packages/ag-charts-website/src/content/docs/layout/index.mdoc

```diff
@@ -7,26 +7,41 @@ This section explains how the chart and its components are sized and laid out wi
 
 ## Chart Size
 
-The chart will auto-size by default, taking the size of the `container` element and auto-sizing the chart dynamically.
-Use the `width` and/or `height` options if a fixed size is required.
+{% if isFramework("javascript") %}
+The chart will auto-size to fill the size of the `container` element. The `class` and `style` attributes are preserved on the `container` element, each of which can be adjust the container's size. Additionally, the container can be placed as a child of a `grid` layout, and the grid will adjust the container's size.
```

Comment:
I feel that:
> The `class` and `style` attributes are preserved on the `container` element, each of which can be adjust the container's size.

...is misleading for JS/TS cases as it implies we do adjust the DOM of the `container`, which is not the case.